### PR TITLE
add upstart service for slack

### PR DESCRIPTION
**Pull request by ipa-fxm**
as requested in https://github.com/unity-robotics/msh/pull/259#discussion_r182474134

@ipa-bnm FYI

as discussed:
this service only makes sense as soon as the `msh`-user is set up
thus, we should add a sub-step to the `PostInstall` during the step where the `msh` user is set up...

### CODE REVIEW

**Comment by ipa-fxm** [comment]
instructions how to install the service has been documented in the `msh_tools/slack/Readme.md` in https://github.com/unity-robotics/msh/pull/259

File: upstart/slack.service [hunk 1]
```diff
@@ -0,0 +1,17 @@
+[Unit]
+Description=service handling slack connection
+Wants=network-online.target
+After=network.target network-online.target NetworkManager.service NetworkManager-wait-online.service
+
+[Service]
+Type=simple
+RemainAfterExit=no
+ExecStart=/bin/bash -c "/usr/bin/tmux -2 new-session -d -s slack && \
+    /usr/bin/tmux -2 send -t slack 'python `rospack find slack`/scripts/ros_service.py'
```
Comment (ipa-fxm) [inline upstart/slack.service]: this script needs [`sys.args`](https://github.com/unity-robotics/msh/pull/259/files#diff-d119f6fdebb87289a5673dc186d04aadR195) which represent [`token`, `channel`, `username`](https://github.com/unity-robotics/msh/pull/259/files#diff-d119f6fdebb87289a5673dc186d04aadR29)

those arguments are robot specific and need to be adjusted before this service can be installed!

Comment (ipa-bnm) [inline upstart/slack.service]: Command Line params missing for der ros_service.py. They need to be: `token`, `channel`, `username`. We should use placeholders e.g. `my_token`, `my_channel`. The placeholders should then be replaced by StickInstall, Postinstall or Userinstall

**Comment by ipa-fxm** [comment]
merge together with #218 
sub-step for PostInstall.sh is still missing

**Comment by ipa-fxm** [comment]
@ipa-bnm 
I messed up here....I pushed to the branch and it did overwrite your fixed systemd service as well as removed the upstart service...
Do you still have it on your laptop?

**Comment by ipa-fxm** [comment]
puuuuh...this saved me :wink: 

```
git reflog show remotes/origin/systemd_slack_service
git reset --hard remotes/origin/systemd_slack_service@{1}
```

**Comment by ipa-fxm** [comment]
will be merged in https://github.com/mojin-robotics/setup_cob4/pull/1

**Pull request closed without merging**